docs/recipes/Theming-Unistyles.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: Theming Ignite with Unistyles
+description: Learn how to use different styling libraries to theme your Ignited app!
+tags:
+  - Theming
+  - iOS
+  - Android
+  - colors
+  - darkmode
+  - unistyles
+last_update:
+  author: Mark Rickert
+publish_date: 2024-12-02
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Theming Ignite
+
+When it comes to styling we acknowledge the popularity and effectiveness of libraries like `styled-components`, `emotion.js` and `unistyles`. Our boilerplate is designed to work seamlessly with these styling solutions, offering you the flexibility to choose the one that aligns with your preferences and project requirements.
+
+The theming system in Ignite Boilerplate is crafted to be adaptable and easy to customize. By simply replacing colors and fonts through the designated theme files, you can tailor the look and feel of your application.
+
+## Using `Unistyles`
+
+:::warning
+
+Unistyles includes custom native code, which means it does not support Expo Go. You'll need to use expo CNG to build your app.
+
+To do this with an newly ignited app, run `yarn expo prebuild --clean` and then `yarn start`.
```

Comment:
since we're saying a "newly ignited app", the command can just be `yarn prebuild:clean`